config/config.go

```diff
@@ -384,6 +386,7 @@ func NewConfig() (c *Config) {
 	c.ExternalServices.Tracing.Path = ""
 	c.ExternalServices.Tracing.InClusterURL = strings.TrimSpace(getDefaultString(EnvTracingInClusterURL, ""))
 	c.ExternalServices.Tracing.URL = strings.TrimSpace(getDefaultString(EnvTracingURL, ""))
+	c.ExternalServices.Tracing.NamespaceSelector = getDefaultBool(EnvTracingNamespaceSelector, true)
```

Comment:
This line is now changed in the latest master. You can actually delete this. If the unit tests need this "true" value to be defined in the default Config object created by NewConfig() function, you can add it in the the NewConfig() function, but for the most part, we don't need to do this anymore because the operator will put this in the ConfigMap explicitly and thus we know it will always be set.